### PR TITLE
Changed the case for starting of comments in html-duration-picker.js

### PR DESCRIPTION
**Pull request by ikayz**

### CODE REVIEW

File: src/html-duration-picker.js [hunk 1]
```diff
@@ -23,38 +23,38 @@
   // and selects the entire block
 
   const selectFocus = (event) => {
-    // get cursor position and select nearest block;
+    // Get cursor position and select nearest block;
```
Comment (nadchif) [inline src/html-duration-picker.js]: Please delete the `;` at the end of the line

File: src/html-duration-picker.js [hunk 2]
```diff
@@ -23,38 +23,38 @@
...
     const cursorPosition = event.target.selectionStart;
     const hourMarker = event.target.value.indexOf(':');
     const minuteMarker = event.target.value.lastIndexOf(':');
 
     if (hourMarker < 0 || minuteMarker < 0) {
-      // something wrong with the format. just return;
+      // Something wrong with the format. just return;
```
Comment (nadchif) [inline src/html-duration-picker.js]: Please delete the `;` at the end of the line

File: src/html-duration-picker.js [hunk 3]
```diff
@@ -76,21 +76,21 @@
...
       return;
     }
-    inputBox.selectionStart = minuteMarker + 1; // seconds mode
+    inputBox.selectionStart = minuteMarker + 1; // Seconds mode
     inputBox.selectionEnd = minuteMarker + 3;
     return;
   };
 
-  // increase time value;
+  // Increase time value;
```
Comment (nadchif) [inline src/html-duration-picker.js]: Please delete the `;` at the end of the line

File: src/html-duration-picker.js [hunk 4]
```diff
@@ -111,7 +111,7 @@
     highlightIncrementArea(inputBox, adjustmentFactor);
   };
 
-  // decrease time value;
+  // Decrease time value;
```
Comment (nadchif) [inline src/html-duration-picker.js]: Please delete the `;` at the end of the line

File: src/html-duration-picker.js [hunk 5]
```diff
@@ -160,32 +160,32 @@
   };
 
   const handleKeydown = (event) => {
-    // use arrow keys to increase value;
+    // Use arrow keys to increase value;
```
Comment (nadchif) [inline src/html-duration-picker.js]: Please delete the `;` at the end of the line

**Comment by ikayz** [comment]
@nadchif sure thing, I'll delete them.

**Comment by nadchif** [comment]
@ikayz How are the changes coming up so far?

**Comment by nadchif** [comment]
I am closing this PR as file has since been refactored and comments already fixed

**Pull request closed without merging**